--- configure.orig	2016-08-24 17:02:51.000000000 +0200
+++ configure	2016-09-22 21:38:46.000000000 +0200
@@ -7050,14 +7050,12 @@
         accept4 \
         backtrace \
         backtrace_symbols_fd \
-        clock_gettime \
 	eventfd \
 	explicit_bzero \
 	timingsafe_memcmp \
         flock \
         ftime \
         getaddrinfo \
-        getentropy \
         getifaddrs \
         getpass \
         getrlimit \
