--- configure.ac.orig	2016-03-25 12:37:28.000000000 +0100
+++ configure.ac	2016-03-25 12:39:31.000000000 +0100
@@ -609,6 +609,7 @@
 
 TOR_SEARCH_LIBRARY(openssl, $tryssldir, [-lssl -lcrypto $TOR_LIB_GDI],
     [#include <openssl/rand.h>],
+    [#include <stdlib.h>],
     [void RAND_add(const void *buf, int num, double entropy);],
     [RAND_add((void*)0,0,0); exit(0);], [],
     [/usr/local/openssl /usr/lib/openssl /usr/local/ssl /usr/lib/ssl /usr/local /usr/athena /opt/openssl])
@@ -686,6 +687,7 @@
 
 TOR_SEARCH_LIBRARY(zlib, $tryzlibdir, [-lz],
     [#include <zlib.h>],
+    [#include <stdlib.h>],
     [const char * zlibVersion(void);],
     [zlibVersion(); exit(0);], [--with-zlib-dir],
     [/opt/zlib])
